conditionals.py

```diff
@@ -105,7 +142,22 @@
 # and print out the corresponding day of the week. 
 # Make sure to add a statement that accounts for any numbers out of range! 
 
+Question = int(input("What day of the week are we in from numbers 1-7? -----> "))
```

Comment:
Variable names should not start with capital letters. 

```suggestion
question = int(input("What day of the week are we in from numbers 1-7? -----> "))
```